internal/build/build.go

```diff
@@ -57,6 +58,22 @@ var (
 	StagingMasAuthURL           = "https://identity.api.stage.openshift.com/auth/realms/rhoas"
 )
 
+// GetEmojiSuccess returns check mark emoji on every platform, but Windows
+func GetEmojiSuccess() string {
+	if runtime.GOOS != "windows" {
+		return "\u2714\ufe0f"
+	}
+	return ""
+}
+
+// GetEmojiError returns cross mark emoji on every platform, but Windows
+func GetEmojiError() string {
+	if runtime.GOOS != "windows" {
+		return "\u274c"
+	}
+	return "Error:"
+}
```

Comment:
Nice! However I think this should not be in the `build` package.

I see us adding some more emojis in the future so a package called `icon` in `pkg` for this would be ideal.

I also think we may want to add one-off emojis sometimes so add a function called `Emoji()` with a default value as the second argument. `Success()` and `Error()` functions can then use this method and you won't need to repeatedly check if runtime.GOOS == windows.

Reply:
@craicoverflow something like this?

`package icon

import "runtime"

func Emoji() bool {
	if runtime.GOOS != "windows" {
		return true
	}
	return false
}

// Success returns check mark emoji on every platform, but Windows
func Success() string {
	if Emoji() {
		return "\u2714\ufe0f"
	}
	return ""
}

// Error returns cross mark emoji on every platform, but Windows
func Error() string {
	if Emoji() {
		return "\u274c"
	}
	return "Error:"
}
`

Reply:
Yeah something like that, but make `Emoji()` function private and call it something else like `hasEmojiSupport`

Reply:
You should also have a generic function that you would use like this:

```go
prefix := icon.Emoji("\u274c", "Error:")

This function should be used by `Error()` and `Success()`

```